processors/piifilterprocessor/processor.go

```diff
@@ -29,23 +29,13 @@ func (p *piiFilterProcessor) ProcessTraces(ctx context.Context, td pdata.Traces)
 	rss := td.ResourceSpans()
 	for i := 0; i < rss.Len(); i++ {
 		rs := rss.At(i)
-		if rs.IsNil() {
```

Comment:
@pavolloffay I am curious about this removal. Is this because of the bump?

Reply:
yes, it was removed from the API.